connect/eaas/core/decorators.py

```diff
@@ -121,5 +122,25 @@ def wrapper(cls):
     return wrapper
 
 
+def transformation(name, description, edit_dialog_ui):
+    def wrapper(cls):
+        if hasattr(cls, TRANSFORMATION_ATTR_NAME):
+            declared_transf = getattr(cls, TRANSFORMATION_ATTR_NAME)
+            if len(list(filter(lambda x: x['name'] == name, declared_transf))) == 0:  # noqa: B023
+                declared_transf.append({
+                    'name': name,
+                    'description': description,
+                    'edit_dialog_ui': edit_dialog_ui,
+                })
+            return cls
+        setattr(cls, TRANSFORMATION_ATTR_NAME, [{
```

Comment:
It should be just an object not an list of objects.
The runner will iterate over all the classes within a module (the module to which we will refer using setuptools entrypoints) that inherit from TransformationBase to build the list of all transformations